doc/source/whatsnew/v0.23.1.txt

```diff
@@ -84,4 +85,4 @@ Reshaping
 
 Other
 
-- Tab completion on :class:`Index` in IPython no longer outputs deprecation warnings (:issue:`21125`)
```

Comment:
Why is this showing up in the diff?

Reply:
Yeah, just noticed the same. Should not be there, sry

Reply:
No worries!  Merging branches can surprise you sometimes 😄

Reply:
there was a removed newline, fixed

Reply:
Great, thanks! I did not have change to fix it during the weekend, but great that it was resolved already.